tests/test_config.py

```diff
@@ -4,14 +4,14 @@
 
 """Test config module."""
 
-from mots.config import calculate_hashes, FileConfig
+from mots.config import calculate_hashes, FileConfig, check_nested_keys_for_value
 
 
 def test_calculate_hashes(config):
     export = b""
     hashes = calculate_hashes(config, export)[1]
 
-    assert hashes["config"] == "6ef5f3ed90c5d9aa2eec7b91ed65a78b886e8fa1"
+    assert hashes["config"] == "76997b8d70e561c7ec8a21e1cefbc3c658c6d91a"
```

Comment:
This is out of scope, but if this hash needs to changed every time something is added to the test config we might want to add an assert message/comment/something to indicate that's why the test has failed.

Reply:
Added note. The export currently doesn't need this since we aren't actually testing the full export content.